### 分叉期间的sector会进入ComputeProofFailed
sealer.log 判读链是否出现分叉， log中会出现commit Seed changed。 
短暂的分叉后， 又会回到主链， 但分叉开始到分叉结束期间的worker 成功做完的sector, 虽然在数据库表里看到1到16的任务都成功了。 
但16完成之后，即commuting状态结束了， 交给sealer的时候， 因为现在是分叉期间， 所以sealer会出现ComputeProofFailed错误。 一般sector错误率控制在1%以内。 

```
2020-07-09T03:41:42.019+0800    ^[[33mWARN^[[0m sectors ffsm/fsm.go:352 planCommitting: got SectorSeedReady, but the seed didn't change
2020-07-09T03:41:42.019+0800    ^[[33mWARN^[[0m sectors ffsm/fsm.go:352 planCommitting: got SectorSeedReady, but the seed didn't change
2020-07-09T03:41:42.019+0800    ^[[33mWARN^[[0m sectors ffsm/fsm.go:352 planCommitting: got SectorSeedReady, but the seed didn't change
2020-07-09T03:41:42.019+0800    ^[[33mWARN^[[0m sectors ffsm/fsm.go:355 planCommitting: commit Seed changed
2020-07-09T03:41:42.019+0800    ^[[34mINFO^[[0m sectors ffsm/fsm.go:212 Sector 944 update state: Committing ...
2020-07-09T03:41:42.043+0800    ^[[34mINFO^[[0m sectors ffsm/states_sealing.go:260      scheduling seal proof computation...
2020-07-09T03:41:42.043+0800    ^[[34mINFO^[[0m sectors ffsm/states_sealing.go:262      KOMIT 944 5cfbe5306fda3337a4eb3bfccd58c100d07587793abef8c5bf625c645853de9b(8209); d84b8b756e9df61714773a71f76fbce5b658a41e9acfc7c03e9726eb3c09b823(9209); [{536870912 bafk4chzahfla46ytve5qpisd7utsb75hzm7b2lsqlkzwfhtz6rrrgujm3ida}]; r:6261666b3465687a61337363707862647367676c326366696f6b796d36776f33646436637168346f363637743772706f73776762756735617132767771; d:6261666b3463687a6168666c6134367974766535717069736437757473623735687a6d3762326c73716c6b7a776668747a3672727267756a6d33696461
2020-07-09T03:41:42.043+0800    ^[[34mINFO^[[0m remote-sealer   remote/sealer.go:617    CommitPhase1 for sector 944 start ...
2020-07-09T03:41:42.050+0800    ^[[34mINFO^[[0m remote-sealer   remote/sealer.go:886    task [{CommitPhase2 {1014 943}}] result-status: 0
2020-07-09T03:41:42.050+0800    ^[[34mINFO^[[0m remote-sealer   remote/sealer.go:988    CommitPhase2 for sector 943 elapsed 1m30.087402454s
2020-07-09T03:41:42.056 INFO filcrypto::proofs::api > verify_seal: start
2020-07-09T03:41:42.056+0800    ^[[33mWARN^[[0m remote-sealer   remote/sealer.go:700    task {CommitPhase1 {1014 944}} has exist ...
2020-07-09T03:41:42.056 INFO filecoin_proofs::api::seal > verify_seal:start
2020-07-09T03:41:42.056 INFO filecoin_proofs::caches > trying parameters memory cache for: STACKED[536870912]-verifying-key
2020-07-09T03:41:42.056 INFO filecoin_proofs::caches > found params in memory cache for STACKED[536870912]-verifying-key
2020-07-09T03:41:42.056 INFO filecoin_proofs::api::seal > got verifying key (536870912) while verifying seal
2020-07-09T03:41:42.056+0800    ^[[33mWARN^[[0m remote-sealer   remote/sealer.go:709    task %v meet the situation which seed changed during doing p34, will reset these work
2020-07-09T03:41:42.067+0800    ^[[33mWARN^[[0m remote-sealer   remote/sealer.go:711    failed add CommitPhase1 task for {CommitPhase1 {1014 944}}: call failed with code=1, msg=Error 1054: Unknown column 'params' in 'field list'
2020-07-09T03:41:42.067+0800    ^[[34mINFO^[[0m remote-sealer   remote/sealer.go:988    CommitPhase1 for sector 944 elapsed 24.305294ms
2020-07-09T03:41:42.067+0800    ^[[34mINFO^[[0m sectors ffsm/fsm.go:212 Sector 944 update state: ComputeProofFailed ...
2020-07-09T03:41:42.077+0800    ^[[34mINFO^[[0m sectors ffsm/fsm.go:145 terminal pledge sector 944
2020-07-09T03:41:42.077+0800    ^[[34mINFO^[[0m remote-sealer   remote/sealer.go:943    sector 944 finished ...
2020-07-09T03:41:42.088 INFO filecoin_proofs::api::seal > verify_seal:finish
2020-07-09T03:41:42.088 INFO filcrypto::proofs::api > verify_seal: finish
2020-07-09T03:41:42.088+0800    ^[[34mINFO^[[0m sectors ffsm/fsm.go:212 Sector 943 update state: CommitOnChain ...
2020-07-09T03:41:42.089+0800    ^[[34mINFO^[[0m sectors ffsm/fsm.go:154 try to cleanup for termnated sector 944, err=<nil>
2020-07-09T03:41:42.089+0800    ^[[34mINFO^[[0m sectors ffsm/fsm.go:170 TerminateState sector 944 entered termnate state ComputeProofFailed, pledging 20 dealing 0
2020-07-09T03:41:42.116+0800    ^[[34mINFO^[[0m sectors ffsm/states_force.go:53 ProveCommitSector msg for sector 943: bafy2bzacecntf3ol6prmcgujzjjw6o4pak2io4246p4g2hzszrgzpxbqlt4bm
2020-07-09T03:41:42.117+0800    ^[[34mINFO^[[0m sectors ffsm/fsm.go:212 Sector 943 update state: CommitWait ...
```

在看：
```
2020-07-09T03:58:16.270+0800    ^[[34mINFO^[[0m sectors ffsm/states_sealing.go:96       performing sector replication...        {"sector": "993"}
2020-07-09T03:58:16.270+0800    ^[[34mINFO^[[0m sectors ffsm/states_sealing.go:98       the sector 993 is a new sector, will get ticket from chain
2020-07-09T03:58:16.284+0800    ^[[34mINFO^[[0m sectors ffsm/fsm.go:212 Sector 993 update state: PreCommit1 ...
2020-07-09T03:58:16.300+0800    ^[[34mINFO^[[0m sectors ffsm/states_sealing.go:96       performing sector replication...        {"sector": "993"}
2020-07-09T03:58:16.300+0800    ^[[34mINFO^[[0m remote-sealer   remote/sealer.go:408    PreCommitPhase1 for sector 993 start ...
2020-07-09T03:58:16.309+0800    ^[[33mWARN^[[0m remote-sealer   remote/sealer.go:473    unable to poll task status for {PreCommitPhase1 {1014 993}}: call failed with code=1, msg=record not found
2020-07-09T03:58:16.816+0800    ^[[34mINFO^[[0m remote-sealer   remote/sealer.go:886    task [{CommitPhase2 {1014 961}}] result-status: 0
2020-07-09T03:58:16.816+0800    ^[[34mINFO^[[0m remote-sealer   remote/sealer.go:988    CommitPhase2 for sector 961 elapsed 1m30.080852789s
2020-07-09T03:58:16.821+0800    ^[[34mINFO^[[0m sectors ffsm/fsm.go:212 Sector 961 update state: CommitFailed ...
2020-07-09T03:58:16.845+0800    ^[[31mERROR^[[0m        sectors ffsm/states_failed_force.go:114 seed changed, will retry: seed epoch doesn't match on chain info: 9273 != 9292
2020-07-09T03:58:16.845+0800    ^[[34mINFO^[[0m sectors ffsm/fsm.go:145 terminal pledge sector 961
2020-07-09T03:58:16.845+0800    ^[[34mINFO^[[0m remote-sealer   remote/sealer.go:943    sector 961 finished ...
```